bundles/AdminBundle/Resources/public/js/pimcore/object/tree.js

```diff
@@ -996,7 +996,7 @@ pimcore.object.tree = Class.create({
             elementType: "object",
             elementSubType: record.data.type,
             id: record.data.id,
-            default: record.data.text
+            default: Ext.util.Format.htmlDecode(record.data.text)
```

Comment:
@BlackbitNeueMedien we should do the same for assets ( https://github.com/pimcore/pimcore/blob/23e82ab797a26e3b5837b622ecbd6b64c5e74604/bundles/AdminBundle/Resources/public/js/pimcore/asset/tree.js#L1259 ) and documents ( https://github.com/pimcore/pimcore/blob/d75f5a1c2faf16ac09c8697f07d78a6e3ca9122d/bundles/AdminBundle/Resources/public/js/pimcore/document/tree.js#L1499 ) right?

Reply:
@brusch Done for assets (I was not aware that "&" is valid in asset keys but in fact it is). For documents & gets replaced by "-", so I think we do not need it there, do we?